cli/src/install.sh

```diff
@@ -131,6 +132,21 @@ copy_files() {
     (umask 077; mkdir -p "${data_dir}")
     cp -a "completions" "${data_dir}/"
     success "Copied completions to ${completions_dir}"
+
+    # Copy extensions over
+    (umask 077; mkdir -p "${extensions_dir}")
+    for ext in extensions/*/PhylumExt.toml
+    do
+        ext_name=$(basename "$(dirname "${ext}")")
+        dest="${extensions_dir}/${ext_name}"
+
+        # Delete the old version if it exists
+        [ -e "${dest}" ] && rm -rf "${dest}"
```

Comment:
This is the reason why I mentioned in the original issue that we might want to hold until we have an idea of what our `upgrade` command should look like.

Do we just want it to delete && install? IIRC one of the main reasons why @andreaphylum didn't want to have install implicitly overwrite the existing version because users might have made local changes. This would apply here too. We'd just nuke anything the user might have changed.

Reply:
Users shouldn't be making changes directly in these folders. But it is definitely true that someone may have installed a modified version of one of our extensions or created their own extension of the same name.

Maybe we should install our extensions into a separate folder?

Reply:
I mean then users just can't make modifications to our extensions at all anymore. That seems very counter-productive.

Extensions are all about hacking some stuff together to fit your own usecase. People should be encouraged to modify the stock extensions, not blocked from doing so.

Reply:
You could argue that someone modifying the extensions should have a local for of our git repository available, though at that point I don't see why `install` wouldn't overwrite too. Either way I'd just be consistent?

Reply:
I believe the current code strikes a decent balance here. Users can install an extension with the same name as an official extension if they want to use a modified version.

Any extensions in the regular extensions directory are used first. And the official extension is only used if no extension exists with the same name.

Reply:
I **really** don't like the idea of creating separate directories for official and unofficial extensions. That just sounds like a cheap hack avoiding a proper solution to me.

And imo it shows in the code, it just introduces so much unnecessary complexity. It doesn't really go hand in hand with the spirit of extensions imo and seems like it's just a worse way to ship built-in subcommands at that point.